buddypress/groups/index-create.php

```diff
@@ -227,7 +227,10 @@
                         </div>
                         <div class="create-group__input-container create-group__input-container--full">
                             <label class="create-group__checkbox-container" for="agree">
-                                <?php print __("I agree to respect and adhere to Mozilla’s Community Participation Guidelines *", "community-portal"); ?>
+                                <p>
```

Comment:
Lets try to avoid using blank elements and apply classes to the paragraph and anchor tag using the BEM convention the site has been following

Reply:
The anchor tag still requires a class here.  

<img width="832" alt="Screen Shot 2020-01-22 at 8 10 52 PM" src="https://user-images.githubusercontent.com/2521244/72948511-5a9c6400-3d53-11ea-95a4-1a5baf2658ab.png">

Additionally can we style the anchor tags in Basic Information, Terms & Responsibilities and the link for the checkbox, to style these they won't have any classes because they come from the post content so in these cases you can select the anchor tag inside the create-group__terms class.  Colour of the anchor should be $color-blue-50.  You can find this in the _vars.scss file.

Reply:
Looking at the figma file the anchors should also have text-decoration: none